scio-parquet/src/main/scala/com/spotify/scio/parquet/read/ParquetReadFn.scala

```diff
@@ -46,12 +47,11 @@ object ParquetReadFn {
   private val EntireFileRange = new OffsetRange(0, 1)
 }
 
-class ParquetReadFn[T, R](
+class ParquetReadFn[T](
   readSupportFactory: ReadSupportFactory[T],
-  conf: SerializableConfiguration,
-  projectionFn: SerializableFunction[T, R]
```

Comment:
this was a parse function, not a projection one. Remove from API and let user map instead.